app/src/main/res/values/colors.xml

```diff
@@ -76,4 +76,29 @@
   <color name="green_shade_40">#6644A85A</color>
   <color name="green_shade_10">#1A44A85A</color>
   <color name="light_green">#D9F3E7</color>
+  <!-- Avatar Background Color -->
```

Comment:
Nit: ``Colors``

Reply:
Also to keep this consistent with other color comments in this file, it should be 
`<!-- AVATAR BACKGROUND COLORS -->`